lib/node/runner/api/pagequery.go

```diff
@@ -7,11 +7,15 @@ import (
 	"strconv"
 
 	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/errors"
 	"boscoin.io/sebak/lib/node/runner/api/resource"
 	"boscoin.io/sebak/lib/storage"
 )
 
-const DefaultMaxLimit uint64 = 100
+const (
+	DefaultLimit    uint64 = 20
+	DefaultMaxLimit uint64 = 100
```

Comment:
The naming here is inconsistent.
One is the default value for `limit` (`DefaultLimit`), the other is the maximum value for `limit` (`DefaultMaxLimit`).
Since there is no `Default`, it should probably be something like `MaxLimit`.
Also, since `limit` is ambiguous, you could say something like `LimitValue` (so `MaxLimitValue`, `DefaultLimitValue`) to make it more clear it's referring to the parameter name.

Reply:
Ok, I got it.